packages/mobile/src/home/WalletHome.test.tsx

```diff
@@ -54,4 +54,64 @@ describe('Testnet banner', () => {
     expect(tree).toMatchSnapshot()
     expect(showMessageMock).toHaveBeenCalledWith('testnetAlert.1', 5000, null, 'testnetAlert.0')
   })
+  it('Renders when disconnected', async () => {
+    const store = createMockStore({
+      ...storeData,
+      account: {
+        backupCompleted: false,
+      },
+    })
+    const tree = renderer.create(
+      <Provider store={store}>
+        <WalletHome
+          refreshAllBalances={jest.fn()}
+          resetStandbyTransactions={jest.fn()}
+          initializeSentryUserContext={jest.fn()}
+          exitBackupFlow={jest.fn()}
+          setLoading={jest.fn()}
+          showMessage={jest.fn()}
+          hideAlert={jest.fn()}
+          importContacts={jest.fn()}
+          loading={false}
+          appConnected={false}
```

Comment:
I think this value gets overwritten by the value set in the provider store. I don't see the banner in the snapshot.

Reply:
Good catch. Updated to use `createMockStoreAppDisconnected()`.